docs/configuration.v2alpha1.md

```diff
@@ -327,7 +327,7 @@ In the table, `spec.override.nodeAgent.image.name` and `spec.override.nodeAgent.
 | [key].securityContextConstraints.customConfiguration.allowedCapabilities | AllowedCapabilities is a list of capabilities that can be requested to add to the container. Capabilities in this field maybe added at the pod author's discretion. You must not list a capability in both AllowedCapabilities and RequiredDropCapabilities. To allow all capabilities you may use '*'. |
 | [key].securityContextConstraints.customConfiguration.allowedFlexVolumes | AllowedFlexVolumes is a whitelist of allowed Flexvolumes.  Empty or nil indicates that all Flexvolumes may be used.  This parameter is effective only when the usage of the Flexvolumes is allowed in the "Volumes" field. |
 | [key].securityContextConstraints.customConfiguration.apiVersion | APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources |
-| [key].securityContextConstraints.customConfiguration.defaultAddCapabilities | DefaultAddCapabilities is the default set of capabilities that will be added to the container unless the pod spec specifically drops the capability.  You may not list a capabiility in both DefaultAddCapabilities and RequiredDropCapabilities. |
+| [key].securityContextConstraints.customConfiguration.defaultAddCapabilities | DefaultAddCapabilities is the default set of capabilities that will be added to the container unless the pod spec specifically drops the capability.  You may not list a capability in both DefaultAddCapabilities and RequiredDropCapabilities. |
```

Comment:
This file is generated, and this typo unfortunately comes from https://github.com/openshift/api/blob/master/security/v1/types.go#L55

Reply:
Got it, will take the chance to maybe ask there to update.